wgkex/client/netlink.py

```diff
@@ -0,0 +1,100 @@
+import re
+
+from pyroute2 import WireGuard, IPRoute
+from pyroute2.netlink.rtnl import ndmsg
+from typing import Dict, List
+from dataclasses import dataclass
+from datetime import datetime, timedelta
+
+
+@dataclass
```

Comment:
The only reason this isn't frozen is because you want to update the interface names?  Or are there other uses coming that require a (shudder) mutable type?

Reply:
Everything in there needs to be updateable on runtime as all attributes will be read from any message queue.